docker-compose.yaml

```diff
@@ -1,25 +1,24 @@
 version: '3'
 
 services:
-
-  server:
-    container_name: server
+  client:
```

Comment:
I would name this frontend, to match how the others are structured, and to avoid confusion with the actual go client that is not dockerized

Reply:
shall we name it application ??

Reply:
go for browser-app :)